be/src/olap/tablet.cpp

```diff
@@ -68,7 +68,7 @@ Tablet::Tablet(TabletMetaSharedPtr tablet_meta, DataDir* data_dir) :
         _last_base_compaction_failure_millis(0),
         _last_cumu_compaction_success_millis(0),
         _last_base_compaction_success_millis(0),
-        _cumulative_point(kInvalidCumulativePoint) {
+        _cumulative_point(tablet_meta->cumulative_layer_point()) {
```

Comment:
cumulative point in tablet meta has non-sense. 
If will cause another persistence for clone/compaction.
So it's removed from tablet meta when refactoring BE.
You can see it because of compatibility for the old code path.

Reply:
Reverted these changes.